specs.cgi

```diff
@@ -39,7 +39,7 @@ case "$prefix" in
     *) prefix=ensmean_;ensmean_selected=selected;plotname="Forecast anomalies";;
 esac
 
-firstdate=`ls kprep/plots/$dataset/25/ | tr " " "\n" | tail -1`
+firstdate=`ls ../climexp_data/KPREPData/plots/$dataset/25/ | tr " " "\n" | tail -1`
```

Comment:
Hoi Folmer,

ik denk dat het robuster is om naar KPREPData te verwijzen en te zorgen dat er een symlink naar de goede plek is. Ik weet niet waar die data staat, hij kan over meerdere schijven vedeeld zijn (zoals op mijn werkstation), maar die symlink vanuit climexp moet altijd werken.

Reply:
Oke, heb '../climexp_data/' weggehaald.